iree/compiler/Codegen/Common/test/iree_comprehensive_bufferize.mlir

```diff
@@ -114,7 +114,8 @@ func @matmul_fill() {
 //  CHECK-DAG:   %[[M:.+]] = hal.interface.constant.load[0]
 //  CHECK-DAG:   %[[N:.+]] = hal.interface.constant.load[1]
 //  CHECK-DAG:   %[[K:.+]] = hal.interface.constant.load[2]
-//  CHECK-DAG:   %[[LHS:.+]] = hal.interface.binding.subspan set(0) binding(0) type(storage_buffer)
+//  CHECK-DAG:   %[[LHS:.+]] = hal.interface.binding.subspan set(0) binding(0) type(storage_buffer) alignment(32)
```

Comment:
Is there a reason the other interface bindings dont get the alignment?

Reply:
The alignment was not specified in the in the test case. It is only specified for this one.